src/borg/helpers/passphrase.py

```diff
@@ -139,3 +142,28 @@ def __repr__(self):
 
     def kdf(self, salt, iterations, length):
         return pbkdf2_hmac('sha256', self.encode('utf-8'), salt, iterations, length)
+
+    def argon2(
+        self,
+        output_len_in_bytes: int,
+        salt: bytes,
+        time_cost,
+        memory_cost,
+        parallelism,
+        type: Literal[b'argon2i', b'argon2d', b'argon2id']
```

Comment:
hmm, thinking about it again, guess i'ld like str here better than bytes. it's text.

do we need to repeat the "argon2" part in the types?

Reply:
> do we need to repeat the "argon2" part in the types?

Are you saying the set of values should be `{'i', 'd', 'id'}`?

Reply:
yes :)